src/main/java/taxi/model/Driver.java

```diff
@@ -6,6 +6,8 @@ public class Driver {
     private Long id;
     private String name;
     private String licenseNumber;
+    private String login;
+    private String password;
 
     public Driver() {
```

Comment:
we should add `toString()` method to Driver class to display driver info correctly in exception messages(but not add password field to it)

Reply:
Ok, done, thought it was optional.